app/views/analytics/_comments.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="chart-wrapper">
+  <div class="chart-title background-gray">
+    Recent Comments
+  </div>
+  <ul class="scroller">
+    <% Comment.order(created_at: :desc).each do |comment| %>
```

Comment:
Look at how I handled the case thumbnails on the homepage.  That's how you should implement this.

Reply:
Ok thanks. I hadn't gotten to these partials yet.